### PR TITLE
Conformance tests for regular users

### PR DESCRIPTION
**Pull request by TerryHowe**
Add a new target to just run one test and a couple new environment variables so you can adjust the cluster yaml

### CODE REVIEW

**Comment by abhay-krishna** [comment]
@TerryHowe: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
eks-anywhere-cluster-controller-tooling-presubmit | 023d1d151aaf6cae6f35a4d1256804efe3d95877 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere/212/eks-anywhere-cluster-controller-tooling-presubmit/1441089105554313216) | `/test eks-anywhere-cluster-controller-tooling-presubmit`

[Full PR test history](https://prow.eks.amazonaws.com/pr-history?org=aws&repo=eks-anywhere&pr=212). [Your PR dashboard](https://prow.eks.amazonaws.com/pr?query=is:pr+state:open+author:TerryHowe).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: test/framework/e2e.go
Comment (jaxesn) [inline test/framework/e2e.go:30]: Does this need to be defined anywhere in codebuild or is it good to go?

Comment (TerryHowe) [reply test/framework/e2e.go:30]: I does not need to be defined. The default value is on line 297 below

**Review by jaxesn — APPROVED** [review]

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere/pull/212#pullrequestreview-763540989" title="Approved">jaxesn</a>*, *<a href="https://github.com/aws/eks-anywhere/pull/212#" title="Author self-approved">TerryHowe</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-anywhere/blob/main/OWNERS)~~ [TerryHowe,jaxesn]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
